### ISSUE TITLE
Battledome: Fight! sometimes fails to use an ability or weapon

### ISSUE THREAD

**Issue by Septimustache** [w35l3y/userscripts#71]
**Browser:** Chrome v93.0.4577.63<br />
**Plugin:** Tampermonkey v4.13<br />
**Script:** Neopets : Battledome : Fight!<br />
**Accessed url:** http://www.neopets.com/dome/arena.phtml<br />
**Installation url:** https://github.com/w35l3y/userscripts/tree/master/scripts/Neopets_Battledome_Fight!<br />
**Error Message:** unsure<br />
**Obs.:** Problem started roughly two weeks ago. No changes to any settings before the problem started. Resetting opponent-specific and default weapons/abilities did not fix it. Uninstalling and reinstalling the script did not fix it.<br />
**Expected behavior:** The script should consistently input the correct weapons and ability for the current opponent.<br />
**Actual behavior:** The script skips inputting the ability and/or weapons at seemingly random intervals. <br />
**Steps to reproduce:**<br />
 1. -Choose a battledome opponent
 2. -Fight using weapons/abilities specific to the opponent or using default weapons/ability.
 3. -Repeat until the problem occurs.

**Example of code**: (Optional, only if you know what it means)
<pre>CODE HERE</pre>

**Comment by mediterra2** [comment]
Yes, this script acts very weirdly on all my computers.  It will sometimes
work for 2 or 3 cycles, then pause and wait for me to intervene, but it
won't let me finish selecting weapons and automatically clicks the fight
button.  Basically, you can only use it on weaker opponents.

**Comment by w35l3y** [comment]
I am not having the spare time needed to keep some scripts.
Pull requests are always welcome.
